pyproject.toml

```diff
@@ -34,3 +35,7 @@ known-first-party = ["accelerate"]
 exclude = [
     "manim_animations/*"
 ]
+
+[tool.ruff.lint.flake8-tidy-imports.banned-api]
+"os.setenv".msg = "Use os.environ instead"
+"os.getenv".msg = "Use os.environ instead"
```

Comment:
Is this because there are some issues with these functions or just to enforce a uniform way of getting env vars?

Reply:
Also I don't think `os.setenv` is a thing?

Reply:
Heh, yeah, `setenv` isn't a thing, it's `putenv`. Will fix.

But yeah, as the PR description says (I don't blame you, I tend to skim them too in general...), to prefer a single way.